vulcan/lib/server/controllers/vulcan_v2_controller.rb

```diff
@@ -368,6 +410,45 @@ def cluster_latency
     end
   end
 
+  def delete_workspace
```

Comment:
@graft @dtm2451 thoughts around db deletes?

Not sure if you guys have any opinions or conventions around deletes for similar endpoints or just mark objects as deleted.

Reply:
we would delete the workspace, run and config records. Which I think is okay. Those records really just store a lot of metadata about the workspace anyways. And since we seem to all agree that we are deleting the workspace directory - those records would just point to stale filesystem paths / data.

Reply:
maybbewe can at least start with this approach and then if a use case comes up - move to soft deletes